data-raw/england-ltla-aande.R

```diff
@@ -0,0 +1,69 @@
+library(tidyverse)
+library(geographr)
+library(healthyr)
+library(compositr)
+library(sf)
+
+ltla <-
+  boundaries_ltla21 |>
+  st_drop_geometry() |>
+  filter(str_detect(ltla21_code, "^E"))
+
+trust_names <-
+  points_nhs_trusts22 |>
+  st_drop_geometry() |>
+  select(starts_with("nhs_"))
+
+# ---- Attendances over 4 hours ----
+attendances_4hours_trust <- england_trust_accidents_emergency |>
+  mutate(date = parse_date_time(date, orders = "B Y")) |>
+  filter(date >= ymd("2022-05-01"), date <= ymd("2023-04-01")) |>
+  subset(select = c(nhs_trust22_code, attendances_over_4hours, pct_attendance_over_4hours)) |>
```

Comment:
I think at the end of this stage, you have lots of `NaN` values in your data, not `NA` values. Note, these are different. I think these `NaN` values are leaking through into your final computations. Try and replace the `NaN` values with `NA` values.